From a75191aff96a4ab8a24ac3ec73db2daee3d960eb Mon Sep 17 00:00:00 2001
From: Appana Durga Kedareswara Rao <appana.durga.rao@xilinx.com>
Date: Fri, 24 Jan 2020 20:52:20 +0530
Subject: [PATCH 1045/1852] dt-bindings: net: axienet: Update binding doc for
 10G/25G MAC 1588

commit 6edcc4361098db95e86fab800455e4d1d47ece9e from
https://github.com/Xilinx/linux-xlnx.git

This patch updates the device-tree binding doc for
adding support for 10G/25G MAC 1588.

Signed-off-by: Appana Durga Kedareswara Rao <appana.durga.rao@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Radhey Shyam Pandey <radhey.shyam.pandey@xilinx.com>
State: not-upstreamable
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/net/xilinx_axienet.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/net/xilinx_axienet.txt b/Documentation/devicetree/bindings/net/xilinx_axienet.txt
index bad67fc67192..381d064aecc0 100644
--- a/Documentation/devicetree/bindings/net/xilinx_axienet.txt
+++ b/Documentation/devicetree/bindings/net/xilinx_axienet.txt
@@ -56,6 +56,7 @@ Optional properties:
 		  unless the PHY is accessed through a different bus).
 - dma-coherent		: Present if dma operations are coherent.
 - xlnx,eth-hasnobuf	: Used when 1G MAC is configured in non-processor mode.
+- xlnx,rxtsfifo		: Configures the axi fifo for receive timestamping.
 
 Optional properties for connected DMA node:
 - xlnx,addrwidth	: Specify the width of the DMA address space in bits.
-- 
2.31.1

